### PR TITLE
Added Fibonacci Number [Java]

### LINKED ISSUE #444: Fibonacci Number [Java]
#### This issue is about [Fibonacci Number](http://www.geeksforgeeks.org/program-for-nth-fibonacci-number/).

- [X] Issue Title is in form `Algo/DS Name [Language]`.
- [X] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [X] This Algo/DS is worth adding to this repository.
- [X] Assign this issue to me (I want to work on this).
	- [X] I ensure that I am not already assigned to 2 or more issues.
- [X] I will send my PR only after I'm assigned to this issue by any maintainer.
- [X] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

### PR DESCRIPTION
**Pull request by jsroyal**
Fixes #444 

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [X] I have read the [Contribution guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [X] I have followed the [coding guidelines](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#cs) for this project.
- [X] My code follows the [skeleton code structure](https://github.com/iiitv/algos/blob/master/CONTRIBUTING.md#sample).
- [X] This pull request has a descriptive title. For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [X] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

### CODE REVIEW

File: fibonacci_number/FibonacciNumber.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import java.util.*;
+/*
+ *Fibonacci series 1,1,2,3,5,8,13,21...
+ *Find the nth value of series.
+ */
+public class FibonacciNumber {
```
Comment (Monal5031) [inline fibonacci_number/FibonacciNumber.java]: Add a blank line after this.

File: fibonacci_number/FibonacciNumber.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	public static int fibonacciNumber(int n) {
+		int sum = 0;
+		int b = 0;
+		int a = 1;		
+        if (n == 1){
```
Comment (Monal5031) [inline fibonacci_number/FibonacciNumber.java]: Add a whitespace after `)`.

File: fibonacci_number/FibonacciNumber.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        	return 1;
+        }
+		for (int i = 1;i < n;i++){
```
Comment (Monal5031) [inline fibonacci_number/FibonacciNumber.java]: Add a whitespace after `)` and after `;`

File: fibonacci_number/FibonacciNumber.java [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+			sum = a + b;
+			b = a;
+			a = sum;	
+		}
+		return sum;
+	}
+
+	public static void main(String[] args) {
+		Scanner in = new Scanner(System.in);
+		int n = in.nextInt();
```
Comment (Monal5031) [inline fibonacci_number/FibonacciNumber.java]: Hardcode value of n

File: fibonacci_number/FibonacciNumber.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+		System.out.println(fibonacciNumber(n));
```
Comment (Monal5031) [inline fibonacci_number/FibonacciNumber.java]: Also user should be able to give its own, initial conditions of 1st and second number, if not provided.
You can do it this way https://stackoverflow.com/a/23093360

File: largest_sum_contiguous_subarray/largestSumContiguousSubarray.c [hunk 6]
```diff
@@ -1,7 +1,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-int largestSumContinousSubArray(int arr[], int size) {
+int largestSumContinousSubarray(int arr[], int size) {
```
Comment (Monal5031) [inline largest_sum_contiguous_subarray/largestSumContiguousSubarray.c]: Revert this change.

File: largest_sum_contiguous_subarray/largestSumContiguousSubarray.c [hunk 7]
```diff
@@ -21,5 +21,5 @@ int main() {
 	for (int k = 0; k < 10; k++) {
 		array[k] = rand() % 10;
 	}
-	printf("%d", largestSumContinousSubArray(array, 10));
+	printf("%d", largestSumContinousSubarray(array, 10));
```
Comment (Monal5031) [inline largest_sum_contiguous_subarray/largestSumContiguousSubarray.c]: Revert this too.

File: largest_sum_contiguous_subarray/largestSumContiguousSubarray.c (hunk 6 above)
Comment (aashutoshrathi) [inline largest_sum_contiguous_subarray/largestSumContiguousSubarray.c]: @jsroyal Sir, Revert all changes in `largestSumContiguousSubarray.c` in **Branch -`jsroyal:fibonacci`**

**Comment by Monal5031** [comment]
@jsroyal sir fix the travis issues, they are mainly of the indentations.

**Comment by Monal5031** [comment]
@jsroyal Sir for java implementation spaces are used but it seems you are using tabs thats why the travis is failing.

**Comment by jsroyal** [comment]
I follow tabs are over the space!
Here is also a nice [article of about it](http://aviaryan.in/blog/tabs-ftw.html)

**Comment by Monal5031** [comment]
@jsroyal I also prefer tabs over spaces but as the repository is configured java for using spaces we have to follow those guidelines, also the travis is failing because of it.
Also, please resolve conflicts in README.md

**Comment by Monal5031** [comment]
@jsroyal Please resolve conflicts.

**Comment by Monal5031** [comment]
@jsroyal Please fix the travis issues.

File: README.md [hunk 8]
```diff
@@ -16,37 +16,40 @@ Community (college) maintained list of Algorithms and Data Structures implementa
...
-| [Shell Sort](https://en.wikipedia.org/wiki/Shellsort) | | [:white_check_mark:](shell_sort/ShellSort.cpp) | [:white_check_mark:](shell_sort/ShellSort.java) | [:white_check_mark:](/shell_sort/shell_sort.py) | [:white_check_mark:](shell_sort/shell_sort.go) | [:white_check_mark:](shell_sort/shellSort.js) | [:white_check_mark:](shell_sort/ShellSort.cs) |
-| [Sieve of Eratosthenes](https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes) | [:white_check_mark:](sieve_of_eratosthenes/sieveOfEratosthenes.c) | | [:white_check_mark:](sieve_of_eratosthenes/SieveOfEratosthenes.java) | [:white_check_mark:](sieve_of_eratosthenes/sieve_of_eratosthenes.py) | [:white_check_mark:](sieve_of_eratosthenes/sieve_of_eratosthenes.go) | [:white_check_mark:](sieve_of_eratosthenes/sieveOfEratosthenes.js) | |
-| [Sleep Sort](http://www.geeksforgeeks.org/sleep-sort-king-laziness-sorting-sleeping/) | | [:white_check_mark:](sleep_sort/sleep_sort.cpp) | | [:white_check_mark:](sleep_sort/sleep_sort.py) | [:white_check_mark:](sleep_sort/sleep_sort.go) | | | |
+
```
Comment (Monal5031) [inline README.md]: Please remove this extra line, its causing the whole diff.

**Comment by aashutoshrathi** [comment]
Resolve Conflicts

**Comment by jsroyal** [comment]
ok! And also add Hactoberfest label to PR.

**Comment by aashutoshrathi** [comment]
@jsroyal Please convert Tab indentation to Space and remove trailing whitespaces.

File: fibonacci_number/FibonacciNumber.java [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+        if (n == 1) {
+            return 1;
+        }
+        for (int i = 1;i < n;i++) {
+            sum = a + b;
+            b = a;
+            a = sum;
+        }
+        return sum;
+        }
```
Comment (singhpratyush) [inline fibonacci_number/FibonacciNumber.java]: Bad indentation.

File: fibonacci_number/FibonacciNumber.java [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+public class FibonacciNumber {
+
+    public static int fibonacciNumber(int n) {
+        int sum = 0;
+        int b = 0;
+        int a = 1;
+        if (n == 1) {
+            return 1;
+        }
+        for (int i = 1;i < n;i++) {
```
Comment (singhpratyush) [inline fibonacci_number/FibonacciNumber.java]: Space after `;`.
